tests/unittests/Foundation/NSURLTests.m

```diff
@@ -291,4 +291,234 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
     NSString* pathAsString = @"/temp/hello/world/中文/你好.txt";
     NSURL* pathAsURL = [NSURL fileURLWithPath:pathAsString];
     ASSERT_OBJCEQ(pathAsString, pathAsURL.path);
-}
\ No newline at end of file
+}
```

Comment:
oops, I was testing something on the reference platform and apparently forgot to delete this?